website/project/views/node.py

```diff
@@ -856,7 +856,7 @@ def private_link_table(node, **kwargs):
 @must_be_valid_project
 def get_editable_children(auth, node, **kwargs):
 
-    if not node.can_edit(auth):
+    if not node.has_permission(auth.user, 'admin'):
```

Comment:
Right. Admins can, read and write cannot. Steve noted that as a side
effect, but it's desired behavior.

On Wed, Sep 23, 2015 at 3:05 PM, Saman Ehsan <notifications@github.com>
wrote:

> In website/project/views/node.py
> <https://github.com/CenterForOpenScience/osf.io/pull/4342#discussion_r40243938>
> :
>
> > @@ -856,7 +856,7 @@ def private_link_table(node, **kwargs):
> >  @must_be_valid_project
> >  def get_editable_children(auth, node, **kwargs):
> >
> > -    if not node.can_edit(auth):
> > +    if not node.has_permission(auth.user, 'admin'):
>
> @jlcohoon <https://github.com/jlcohoon> I thought only admin users could
> create VOLs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/4342/files#r40243938>
> .
>